app/webpacker/styles/layout.scss

```diff
@@ -31,7 +31,7 @@ main,
   width: 1px;
 }
 
-.hidden {
+.js-enabled .hidden {
```

Comment:
Will changing this selector alter the behaviour of any other controllers that toggle this `hidden` class? (they may have their own selectors dealing with JS enabled/disabled?)

Reply:
No this is safe, it will not change the behaviour of any other controllers.